src/dashboard/Dashboard.spec.js

```diff
@@ -1 +1,102 @@
 // Test away
+import React from "react";
+import ReactDOM from "react-dom";
+import { render, fireEvent } from "react-testing-library";
+import "jest-dom/extend-expect";
+import Controls from "./../controls/Controls";
+import Display from "./../display/Display";
+
+it("Checks if close button is called", () => {
+  const mockFn = jest.fn();
+  const { getByText } = render(<Controls toggleClosed={mockFn} />);
+
+  const closeButton = getByText("Close Gate");
+
+  fireEvent.click(closeButton);
+
+  expect(mockFn).toHaveBeenCalled();
+});
+
+it("Checks if open button is called", () => {
+  const mockFn = jest.fn();
+  const { getByText } = render(
+    <Controls closed={true} toggleClosed={mockFn} />
+  );
+
+  const openButton = getByText("Open Gate");
+
+  fireEvent.click(openButton);
+
+  expect(mockFn).toHaveBeenCalled();
+});
+
+it("Checks if unlock button is called", () => {
+  const mockFn = jest.fn();
+  const { getByText } = render(
+    <Controls closed={true} locked={true} toggleLocked={mockFn} />
+  );
+
+  const unlockButton = getByText("Unlock Gate");
+
+  fireEvent.click(unlockButton);
+
+  expect(mockFn).toHaveBeenCalled();
+});
+
+xit("Checks if lock button is called", () => {
```

Comment:
This test is failing because the other tests' results haven't been cleaned up; here's how you could do that:
```javascript
it("Checks if unlock button is called", () => {
  const mockFn = jest.fn();
  const { getByText } = render(
    <Controls closed={true} locked={true} toggleLocked={mockFn} />
  );

   const unlockButton = getByText("Unlock Gate");

   fireEvent.click(unlockButton);

   expect(mockFn).toHaveBeenCalled();
   // cleans up for next test
   cleanup(); 
});
```
The typical way people will do this is by adding `afterEach(cleanup)` to the specific scope they want it to apply in. In your case, you should be able to do it right in the global scope of this file, up top preferably. Here are the relevant docs for afterEach:
- [Repeating setup and/or teardown for many tests](https://jestjs.io/docs/en/setup-teardown#repeating-setup-for-many-tests)
- [afterEach API reference](https://jestjs.io/docs/en/api#aftereachfn-timeout)